portal/auth.py

```diff
@@ -39,8 +39,8 @@ def index():
                 return "Hello teacher"
                 #return redirect(url_for('portal.teacher_page'))
             elif user['role'] == 'student':
-                return "hello student"
-                #return redirect(url_for('portal.student_page'))
```

Comment:
This comment should be removed. No longer necessary now that the student landing page is created.